dolphinscheduler-task-plugin/dolphinscheduler-task-pytorch/src/main/java/org/apache/dolphinscheduler/plugin/task/pytorch/PytorchParameters.java

```diff
@@ -35,7 +35,7 @@ public class PytorchParameters extends AbstractParameters {
     private String pythonPath = ".";
     private String script;
     private String scriptParams;
-    private String pythonLauncher = "${PYTHON_LAUNCHER}";
+    private String pythonCommand = "${PYTHON_LAUNCHER}";
```

Comment:
You should change the frontend part since `PYTHON_LAUNCHER` is a unified name.

Reply:
As you say, I have revert the first commit and renamed the attribute pythonCommand to pythonLauncher in frontend.